src/ExtInterface/ReefMod/MatDomain.jl

```diff
@@ -0,0 +1,276 @@
+using ADRIA: SimConstants, Domain, GDF
+using AxisKeys
+using DataFrames
+using DimensionalData
+using Distributions
+using MAT
+using ModelParameters
+using NamedDims
+using NetCDF
+using Statistics
+using YAXArrays
```

Comment:
Group like imports together and list alphabetically as in Blue style guide.

e.g.,

```julia
using
    Distributions
    Statistics

using
    AxisKeys
    DimensionalData
    MAT
    NamedDims
    NetCDF
    YAXArrays
```

Incidentally, if DimensionalData is being imported to get `At`, then I'd prefer you import that directly:

```julia
import YAXArrays.DD: At
```